components/feature/addons/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -53,7 +53,7 @@
     <!-- The authors of an add-on. -->
     <string name="mozac_feature_addons_authors">Autores</string>
     <!-- The last date that the add-on was updated. -->
-    <string name="mozac_feature_addons_last_updated">Ultima atualização</string>
+    <string name="mozac_feature_addons_last_updated">Última atualização</string>
```

Comment:
@Pike Android lint was complaining about this on [Task Cluster](https://firefox-ci-tc.services.mozilla.com/tasks/X7fakBMrTsqfogf6xqzd7g/runs/0/logs/https%3A%2F%2Ffirefox-ci-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FX7fakBMrTsqfogf6xqzd7g%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log) Should I keep the change here or just ignore it? 

```
/builds/worker/checkouts/src/components/feature/addons/src/main/res/values-pt-rBR/strings.xml:56: Error: "Ultima" is a common misspelling; did you mean "Última" ? [Typos]
<string name="mozac_feature_addons_last_updated">Ultima atualização</string>
                                                                                         
```

Reply:
I've made the corresponding change in Pontoon now, too. Matches what the translation memory has, that's why.